### ISSUE TITLE
Leiterplatine

### ISSUE THREAD

**Issue by Murdock79** [madmartin/Jarolift_MQTT#64]
Hat jemand eine saubere Leiterplatine entworfen und kann mir die Datei zur Verfügung stellen, damit ich sie mir anfertigen lassen kann?
Selbstgelötet auf Lochrasterplatine sieht es immer etwas frickelling aus.

**Comment by Subbilein** [comment]
Ich hätte auch Interesse :)
Meine Löt-Skills sind auch begrenzt.

**Comment by Murdock79** [comment]
Habe mir das über eine Software konstruiert und in China anfertigen lassen. Habe gleich 5 Platinen gefertigt bekommen. Falls Interesse besteht und die Platinen funktionieren, kannst du gerne eine Platine von mir haben. Muss noch warten, bis sie zu mir geschickt werden.

**Comment by Subbilein** [comment]
Das wäre super.
Kannst du mich dann auch mit dem MasterMSB Key supporten ?
Danke schonmal!

**Comment by Murdock79** [comment]
Ja, kein Problem. Ich helfe dir. Habe sehr lange gebraucht, um das zu checken und gebe mein Wissen gerne weiter. Schreib mir einfach privat.

**Comment by Subbilein** [comment]
Vielen Dank. Wie kann ich dir privat schreiben?

**Comment by Murdock79** [comment]
> Vielen Dank. Wie kann ich dir privat schreiben?

Email sollte nun in meinem github Profil angezeigt werden.

**Comment by Subbilein** [comment]
Wird leider nicht angezeigt. 
dann bitte an mr.subba@web.de

**Comment by Stevelande** [comment]
Habe schon eine laufende Version, dennoch wäre ich Interessiert an der Platine. Hast du etwas mehr Infos? Schaltplan, Komponenten?
Mein Wissen gebe ich auch gerne weiter. stevelandisch@gmail.com

**Comment by Murdock79** [comment]
Bin grad im Urlaub, und werde den Schaltplan hier posten, wenn es funktioniert.
Meine Mail Eintraum79@googlemail.com
Hat jemand die Software mit Platformio installiert und kann mir dabei helfen?
